### PR TITLE
Branding

### PR DESCRIPTION
**Pull request by clusterwin**
Add in netflix branding, remove the tinder, add app icon.

### CODE REVIEW

**Comment by clusterwin** [comment]
demo

**Comment by clusterwin** [comment]
demo

**Comment by clusterwin** [comment]
demo

**Comment by clusterwin** [comment]
@gprasant @greysongregory Fixing up some visual bugs that were bothering me.

File: flixnchill/DraggableImageView.m [hunk 1]
```diff
@@ -50,12 +50,13 @@ -(void) initSubviews {
     self.contentView.layer.borderColor = [[UIColor grayColor] CGColor];
     self.contentView.layer.borderWidth = 1.0f;
     [self.contentView.layer setCornerRadius:10.0f];
-    self.originalCenter = [self.contentView center];
+	self.originalCenter = [self.contentView center];
```
Comment (greysongregory) [inline flixnchill/DraggableImageView.m:53]: accident?

File: flixnchill/LoginViewController.m [hunk 2]
```diff
@@ -20,6 +20,8 @@ @implementation LoginViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
+	UIColor *netflixRed = [UIColor colorWithRed:(185/255.0) green:(9/255.0) blue:(11/255.0) alpha:1] ;
```
Comment (greysongregory) [inline flixnchill/LoginViewController.m:23]: align

File: flixnchill/SettingsViewController.m [hunk 3]
```diff
@@ -22,6 +22,8 @@ @implementation SettingsViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+	UIColor *netflixRed = [UIColor colorWithRed:(185/255.0) green:(9/255.0) blue:(11/255.0) alpha:1] ;
```
Comment (greysongregory) [inline flixnchill/SettingsViewController.m:25]: align

File: flixnchill/CardsViewController.m [hunk 4]
```diff
@@ -32,7 +32,9 @@ @implementation CardsViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    
+
```
Comment (greysongregory) [inline flixnchill/CardsViewController.m:35]: align

File: flixnchill/CardsViewController.m [hunk 5]
```diff
@@ -45,7 +47,7 @@ - (void)viewDidLoad {
             [self initSubViews];
         }
     }];
-    
+	[self.view bringSubviewToFront:self.draggableCard];
```
Comment (greysongregory) [inline flixnchill/CardsViewController.m:50]: align

File: flixnchill/ChatsViewController.m [hunk 6]
```diff
@@ -18,6 +18,8 @@ @implementation ChatsViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
+	UIColor *netflixRed = [UIColor colorWithRed:(185/255.0) green:(9/255.0) blue:(11/255.0) alpha:1] ;
```
Comment (greysongregory) [inline flixnchill/ChatsViewController.m:21]: align

**Comment by greysongregory** [comment]
seems simple enough to me... bunch of alignment issues, but other than that LGTM

**Comment by gprasant** [comment]
LGTM

**Comment by clusterwin** [comment]
Fixed alignment issues and reenabled facebook login.

**Pull request merged**